schemas/derivatives/SCHEMA_AMM (with vault)

```diff
@@ -0,0 +1,116 @@
+| close_fees_usd          | The total close fees accrued for each pair (24h rolling average, in USD). | number |
+
+### LP Snapshot
+
+Liquidity providers snapshot (one entry for each token in a vault).
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the snapshot.                            | number |
+| block_date               | The timestamp truncated (ie, YYYY-MM-DD format for daily snapshots and YYYY-MM-DD HH:00:00 for hourly snapshots). | date |
+| chain_id                 | Standard chain id.                                        | number |
+| vault_address            | The address of the vault.                                  | string |
+| lp_address               | The address of the liquidity provider.                    | string |
+| token_index              | The index of the token in the smart contract.             | number |
+| token_address            | The address of the token provided as liquidity.           | string |
+| token_symbol             | The symbol of the token.                                  | string |
+| amount_useful            | The amount of tokens supplied by the LP that are liquid and usable by users in the vault, decimal normalized. | number |
+| amount_useful_usd        | The amount of liquid tokens supplied in USD. | number |
+| liquidated_amount        | The amount of tokens liquidated from the LP (when the taker has positive PnL if a position is closed), decimal normalized. | number |
+| liquidated_amount_usd    | The amount liquidated, in USD.                            | number |
+| total_lp_fees_usd        | Total fees generated for the LP at the time of snapshot (in USD). | number |
+
+### Vault Snapshot
+
+Snapshot of the vault's metrics.
+
+| Property                | Description                                               | Type   |
+|-------------------------|-----------------------------------------------------------|--------|
+| timestamp                | The timestamp of the snapshot.                            | number |
+| block_date               | The timestamp truncated (ie, YYYY-MM-DD format for daily snapshots and YYYY-MM-DD HH:00:00 for hourly snapshots). | date |
+| chain_id                 | Standard chain id.                                        | number |
+| vault_address            | The smart contract address of the vault (one address for each vault, if you have multiple vaults). | string |
+| token_index              | The index of the tokenin the smart contract (one row for each pair in a vault). | number |
+| tvl                      | Total Value Locked at snapshot, in tokens (one entry for each token in the vault). | number |
+| tvl_usd                  | Total Value Locked at snapshot (TOTAL of all tokens, in USD). | number |
```

Comment:
tvl - is this the token amount ? if so maybe lets name it token_amount and token_amount_usd?

also token_address is missing

Reply:
sounds good, making the changes

Reply:
I find token_amount rather confusing compared to tvl. 
I don't think we need any token_address at the vault level, just the vault address, don't we?

Reply:
using sudo as an example here 

their SLP is one vault, so shouldnt there be token address to identify the correct asset?

Reply:
indeed, i'll add it.